ts-client/src/vault/tests/affiliate.test.ts

```diff
@@ -59,6 +63,10 @@ describe('Interact with Vault in devnet', () => {
         const depositResult2 = await provider.sendAndConfirm(depositTx2);
         console.log('Deposit result', depositResult2);
         expect(typeof depositResult2).toBe('string');
+
+        // Check balance again
+        const userBalanceDeposit2 = await vaultImpl.getUserBalance(mockWallet.publicKey);
+        expect(Number(userBalanceDeposit2)).toBeGreaterThan(0);
```

Comment:
i think here we can check greater than the first amount?

Reply:
yep done both, good suggestion